### PR TITLE
DOC: mentions SSH port forwarding on web ui page

### PR DESCRIPTION
**Pull request by stsievert**
I added a couple paragraphs about what I experienced in #418. I give an example of port forwarding.

### CODE REVIEW

**Comment by mrocklin** [comment]
Cool.  One concern I have is that people may interpret this as the preferred way to reach the web interface, rather than simply pointing their browser to an accessible HTTP route.  If we include this in our documentation then we need to make sure that it is appropriately scoped so that novice users don't trip up.

**Comment by stsievert** [comment]
Addressed. I added a section on the default behavior. I said if that doesn't work, one way to get around it is via port forwarding.

File: docs/source/web.rst [hunk 1]
```diff
@@ -38,6 +38,50 @@ tasks.  For the most recent 20000 tasks visit
...
+.. code:: bash
+
+   local$ ssh -L 8000:localhost:8787 user@remote
+   remote$ dask-scheduler  # now, the web UI is visible at localhost:8000
+   remote$ # continue to set up dask if needed -- add workers, etc
+
+It is then possible to go to ``localhost:8000`` and see Dask Web UI.
+
+This can be used in conjuction with port forwarding on the Jupyter notebook
+port.
```
Comment (mrocklin) [inline docs/source/web.rst]: Why the note here about Jupyter notebooks?

Comment (stsievert) [inline docs/source/web.rst]: Because it's how I'm using distributed and I think it's useful.

Comment (mrocklin) [inline docs/source/web.rst]: Hrm, perhaps generalize to say something like.  "This same approach is not specific to dask.distributed, but can be used by any service that operates over a network, such as Jupyter notebooks."

Alternatively, perhaps you could write a tiny external blogpost about how many python services like Jupyter and Dask are now providing web interfaces and how you can access them through ssh tunneling.  Then we could link to that page from these docs.

There are enough varied users of Dask that operate in such different ways that I want to make sure we don't get pulled in the game of documenting them all.  I'm afraid that things will start to become confusing as novice users have to figure out what documentation is essential and what documentation was particular to a specific way of using the system.

Comment (stsievert) [inline docs/source/web.rst]: I added Jupyter as an example.

> you could write a tiny external blogpost

I've been drafting that post (and filed this issue while preparing to write it). I plan to cover how I use dask with UW–Madison's cluster and my personal setup (web UI + remote Jupyter notebook).

**Comment by mrocklin** [comment]
Merging!  Thanks for getting this in.

**Pull request merged**